.gitmodules

```diff
@@ -99,7 +99,7 @@
 	url = https://github.com/awslabs/aws-c-event-stream
 [submodule "aws-c-common"]
 	path = contrib/aws-c-common
-	url = https://github.com/ClickHouse/aws-c-common
+	url = https://github.com/awslabs/aws-c-common.git
```

Comment:
Why?

Reply:
> our fork no longer needed since cJSON was removed

Reply:
> our fork no longer needed since cJSON was removed

It's still there:
https://github.com/awslabs/aws-c-common/blob/main/source/external/cJSON.c

But I see they disabled those lines from https://github.com/aws/aws-sdk-cpp/pull/2231 which we cherry-picked